plugin/Field/Collection/FieldLoaderGeneric.php

```diff
@@ -77,6 +77,48 @@ public function load(): Generator
 			$module = $moduleProperties['id'];
 			$fieldArray = $moduleProperties['elements'];
 
+			if ($module === "address") {
+				$newFieldArray = [
```

Comment:
@hungnc89 This is out of scope for this class and Should be inside a `FieldModuleDecorator`, basically the way it was. 

Background: many views and classes need different fields than others. Instead of cluttering the code with `if/else` cases, these were put into these `FieldModuleDecorator` classes that could be used wherever needed.
If you need a neat way to make sure the configuration is the same for each caller, you can try to add the configuration to `FieldsCollectionBuilderShort`.